src/Lean/Elab/BuiltinCommand.lean

```diff
@@ -138,6 +173,106 @@ private partial def elabChoiceAux (cmds : Array Syntax) (i : Nat) : CommandElabM
     aliases := aliases.push (currNamespace ++ id, declName)
   modify fun s => { s with env := aliases.foldl (init := s.env) fun env p => addAlias env p.1 p.2 }
 
+/-- Makes names from other namespaces visible without writing the namespace prefix.
+
+This command only makes *public* names visible, *i.e.* it does not bring `protected` and `private`
+names in scope.
```

Comment:
This is not true:
```lean
private def foo.bar := 1

open foo (bar)

#check bar
```

Reply:
We should definitely clear this up.

Reply:
Right, I meant that `open My.Namespace` with no `(Ident1 Ident2 ...)`, consistently with your other remark on this topic. Definitely needs a clarification indeed.